pyvista/core/filters/data_set.py

```diff
@@ -2173,6 +2173,160 @@ def streamlines_from_source(dataset, source, vectors=None,
+            only LENGTH_UNIT (``'l'``) and CELL_LENGTH_UNIT (``'cl'``).
+            Default is CELL_LENGTH_UNIT: ``'cl'``.
+
+        max_steps : int, optional
+            Maximum number of steps for integrating a streamline.
+            Defaults to ``2000``
+
+        terminal_speed : float, optional
+            Terminal speed value, below which integration is terminated.
+
+        interpolator_type : str, optional
+            Set the type of the velocity field interpolator to locate cells
+            during streamline integration either by points or cells.
+            The cell locator is more robust then the point locator. Options
+            are ``'point'`` or ``'cell'`` (abbreviations of ``'p'`` and ``'c'``
+            are also supported).
+
+        separating_distance : float, optional
+            The distance between streamlines expressed in ``step_unit``.
+
+        separating_distance_ratio : float, optional
+            Streamline integration is stopped if streamlines are closer than
+            ``SeparatingDistance*SeparatingDistanceRatio`` to other streamlines.
+            The default behavior is set by vtk.
+
+        closed_loop_maximum_distance : float, optional
+            The distance between points on a streamline to determine a 
+            closed loop.
+
+        loop_angle : float, optional
+            The maximum angle in degrees between points to determine a closed loop.
+
+        minimum_number_of_loop_points : int, optional
+            The minimum number of points before which a closed loop will
+            be determined.
+
+        compute_vorticity : bool, optional
+            Vorticity computation at streamline points. Necessary for generating
+            proper stream-ribbons using the ``vtkRibbonFilter``.
+
+        Returns
+        -------
+        streamlines : pyvista.PolyData
+            This produces polylines as the output, with each cell
+            (i.e., polyline) representing a streamline. The attribute values
+            associated with each streamline are stored in the cell data, whereas
+            those associated with streamline-points are stored in the point data.
+
+        Example
+        -------
+        Plot evenly spaced streamlines for cylinder in a crossflow.
+        This dataset is a multiblock dataset, and the fluid velocity is in the
+        first block.
+
+        >>> import pyvista
+        >>> from pyvista import examples
+        >>> mesh = examples.download_cylinder_crossflow()
+        >>> streams = mesh[0].streamlines_evenly_spaced_2D(start_position=(4, 0.1, 0.),
+        ...                                                separating_distance=3,
+        ...                                                separating_distance_ratio=0.2)
+        >>> plotter = pyvista.Plotter()
+        >>> _ = plotter.add_mesh(streams.tube(radius=0.02), scalars="vorticity_mag")
+        >>> plotter.view_xy()
+        >>> _ = plotter.show()
```

Comment:
```suggestion
        >>> plotter.show()
```
No longer necessary.